scipy/stats/_distn_infrastructure.py

```diff
@@ -40,8 +40,11 @@
 if PY3:
     def instancemethod(func, obj, cls):
         return types.MethodType(func, obj)
+
+    from itertools import zip_longest
```

Comment:
We no longer support Python 2, so this line can be moved out of this `if` block.  That is, we can run `from itertools import zip_longest` unconditionally.  (The clean up of the other conditional import does not have to be done in this pull request.)